### ISSUE TITLE
Pull in Meetup.com meetups

### ISSUE THREAD

**Issue by ChaseDudas** [Denver-Devs/denverdevs.org#135]
**Is your feature request related to a problem? Please describe.**
It would be amazing to start pulling in some meetups to the site

**Describe the solution you'd like**
-- @danhannigan, I will default this question to you off the rip --

**Describe alternatives you've considered**
There are no alternative solutions or features considered yet.

**Additional context**
How was this accomplished on the last site?

**Comment by danhannigan** [comment]
@ChaseDudas previously we did kind of a funky hack where we'd pull in the meetups via a node script, that json response would get stashed prior to the site build, and then we'd use that json in the site. The meetups were also a curated textfile list that other people could contribute to.

At any rate I just went digging and it looks like Meetup has changed their API a good amount, including locking it behind a Pro paywall... so, this idea might be dead in the water. I think it's like $30/mo charged in 6 month intervals, and that's just wild. I still have an old auth key for it, so I'll see if it can work.
